config/initializers/rack_attack.rb

```diff
@@ -21,13 +24,28 @@ class Rack::Attack
   # counted by rack-attack and this throttle may be activated too
   # quickly. If so, enable the condition to exclude them from tracking.
 
+  # Thottle elasticsearch normal queries.
+  #
+  # The regex comes from:
+  # echo $(grep elastic /etc/nginx/conf.d/ao3_routing.conf  | grep -v  "unicorn_elastic_bookmarks" | awk '{print $1}' | cut -c 4- | sed -e 's/"/|/' -e 's#\/#\\/#g' ) | sed -e 's/ //g'
```

Comment:
These comments should be in the issue or pull request description. Most code readers will not have access to ao3_routing.conf anyway.